packages/record-data/tests/test-helper.js

```diff
@@ -0,0 +1,48 @@
+import { setApplication } from '@ember/test-helpers';
```

Comment:
Can you please add a comment that says this is a copy paste (if it is?). Can we make it a symlink instead? Would that be too trolly

Reply:
Had another comment that called this out but yes, testem, test-helper and ember-cli-build file changes are all just copy paste. Since they are all JS files I'm considering re-exporting them from the test-infra package instead so that we can maintain in one place. Was thinking that would be a follow up if it is what we want.